Sources/Subs-Db-mysql.php

```diff
@@ -776,13 +776,24 @@ function smf_db_insert($method = 'replace', $table, $columns, $data, $keys, $ret
 	global $smcFunc, $db_connection, $db_prefix;
 
 	$connection = $connection === null ? $db_connection : $connection;
+	
+	$return_var;
```

Comment:
i need the var across the methode,
to get the right scrope i defined this var on start.